### PR TITLE
Get all for events and Post for order

### PR DESCRIPTION
**Pull request by BuneaAlex**

### CODE REVIEW

File: src/api_calls/utils.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    return response.json()
+}
+
+export function authenticationHeaderSetter(headers)
+{
+    let username = localStorage.getItem('email');
+    let password = localStorage.getItem('password');
+    let encodedCredentials = btoa(username + ':' + password);
+    headers.append('Authorization', 'Basic ' + encodedCredentials);
+}
```
Comment (karinasigartau0798) [inline src/api_calls/utils.js]: Please add new line at the end of the file.

File: main.js [hunk 2]
```diff
@@ -93,17 +94,16 @@ function renderHomePage() {
...
 
   eventData.forEach(event => {
-
     const eventCard = document.createElement('div');
-  eventCard.classList.add('event-card');
-  // Create the event content markup
-  const contentMarkup = `
+    eventCard.classList.add('event-card');
 
+    eventCard.setAttribute('data-event', JSON.stringify(event));
```
Comment (karinasigartau0798) [inline main.js]: Why is this needed?

File: src/api_calls/orders_calls.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    return fetch(JAVA_SERVER_ORDERS_BASE_URL,options)
+        .then(status)
+        .then(response=>{
+            return response.text();
+        }).catch(error=>{
+            console.log('Request failed', error);
+            return Promise.reject(error);
+        });
+}
```
Comment (karinasigartau0798) [inline src/api_calls/orders_calls.js]: Please add a new line at the end of the file.

File: main.js (hunk 2 above)
Comment (BuneaAlex) [reply main.js]: Nu e, dar am sters-o.

File: src/api_calls/orders_calls.js (hunk 3 above)
Comment (BuneaAlex) [reply src/api_calls/orders_calls.js]: Am pus la toate cred, la ultimul commit.

File: main.js [hunk 4]
```diff
@@ -1,3 +1,6 @@
+import { data } from "autoprefixer";
```
Comment (karinasigartau0798) [inline main.js]: Is "data" used anywhere? If not, you can remove the import.

Comment (BuneaAlex) [reply main.js]: Cred ca era folosit in proiectul initial, dar eu nu-l mai folosesc. Am sters toate importurile pe care nu le folosesc.

**Review by karinasigartau0798 — APPROVED** [review]
